src/oumi/cli/infer.py

```diff
@@ -67,13 +80,21 @@ def infer(
     Args:
         ctx: The Typer context object.
         config: Path to the configuration file for inference.
+        output_dir: Directory to save configs
+        (defaults to OUMI_DIR env var or ~/.oumi/configs).
         interactive: Whether to run in an interactive session.
         image: Path to the input image for `image+text` VLLMs.
         system_prompt: System prompt for task-specific instructions.
         level: The logging level for the specified command.
     """
     extra_args = cli_utils.parse_extra_cli_args(ctx)
 
+    if config:
+        if config.startswith("oumi://"):
+            _ = cli_utils.resolve_and_fetch_config(config, output_dir)
+            cleaned_path, config_dir = cli_utils.resolve_oumi_prefix(config, output_dir)
+            config = str(config_dir / cleaned_path)
```

Comment:
With my suggested changes to let `resolve_and_fetch_config` handle all prefixes, we can simplify this:

```suggestion
        config = cli_utils.resolve_and_fetch_config(config, output_dir)
```